test_system.py

```diff
@@ -0,0 +1,66 @@
+import os
+import pytest
+import app
+from trello_items import create_board, delete_board
+from threading import Thread
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+import dotenv
+import requests
+ 
+@pytest.fixture(scope='module')
+def test_app():
+
+    file_path = dotenv.find_dotenv('.env')     
+
```

Comment:
You should have this line after setting file_path (currently .env is not loaded after you find it):

`dotenv.load_dotenv(file_path, override=True)`

But this file does pass when running it individually in an IDE that loads .env automatically